src/language-js/printer-estree.js

```diff
@@ -1088,9 +1088,24 @@ function printPathNoParens(path, options, print, args) {
       parts.push("#");
       return parts;
     }
-
     case "ArgumentPlaceholder":
       return "?";
+    case "ModuleExpression": {
+      const isNonEmptyBody = isNonEmptyArray(n.body.body);
+      parts.push("module {");
+      if (isNonEmptyBody) {
+        parts.push(indent([hardline, path.call(print, "body")]), hardline);
+      } else {
+        if (hasComment(n, CommentCheckFlags.Dangling)) {
+          parts.push(
+            printDanglingComments(path, options /** sameIndent */),
```

Comment:
Why don't we add comment as `Program`'s dangling comment, it's already handled.

Reply:
You're right.. I'm not sure why I chose it..